.github/workflows/validator-update.yml

```diff
@@ -32,7 +32,8 @@ jobs:
       - name: Create task to run cloud function
         run: |
           gcloud tasks create-http-task \
-            --queue=update_validation_report_task_queue \
+            --queue=update-validation-report-task-queue \
```

Comment:
I think having 2 ways to name the queue in [main.tf](https://github.com/MobilityData/mobility-feed-api/blob/77b7811af03506147b6ebda49cce11e053fb115d/infra/functions-python/main.tf#L613C1-L618C2) (with underscore and dash) is just confusing. Why can't both be the same?

Reply:
Currently, all our resources are named using underscores, like `name_of_resource`, whereas individual elements are named with hyphens, such as `name-of-element` (this naming convention also applies to Python functions). However, when I try to name the queue as `name_of_queue`, GCloud throws an [error](https://github.com/MobilityData/mobility-feed-api/actions/runs/11709863979/job/32614820434).
The gcloud system requires queue IDs to contain only letters, numbers, or hyphens (`-`), and restricts the use of underscores.

Reply:
FYI there's at least one case where the undercrore is used in both  "names" (See [there](https://github.com/MobilityData/mobility-feed-api/blob/77b7811af03506147b6ebda49cce11e053fb115d/infra/workflows/main.tf#L62)). But it's not a queue.

What about naming them differently? AFAIK the name on the resource line (update_validation_report_task_queue) is used only within terraform. So we could have some kind of suffix.

In any case I guess this might be out of scope for this PR.

Reply:
Good catch! I guess it would be valuable to go over the naming convention in terraform. Issue created: https://github.com/MobilityData/mobility-feed-api/issues/802 -- feel free to add some details.